src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+function addMessage(messageText, isError = false) {
+  const message = document.createElement('div');
+
+  message.textContent = messageText;
+  message.classList.add('message', isError ? 'error-message' : null);
+
+  document.body.append(message);
+}
+
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => resolve());
+});
+
+promise1.then(() => addMessage('Promise was resolved!'));
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(reject, 3_000);
```

Comment:
It's a good practice to provide an error message or reason when rejecting a promise. Consider passing an error message to `reject`, like `reject('Timeout occurred')`, to make debugging easier.